courses/fundamentals_of_ada/adv_140_access_types.rst

```diff
@@ -128,6 +128,105 @@ Access Types and Primitives
 
          procedure Proc (V : access T); -- Primitive of T
 
+-----------------------
+Anonymous Access Types
+-----------------------
+
+* Can be declared in several places
+
+    + Are **pool-specific**
```

Comment:
Anonymous are general, not pool-specific

Reply:
Indeed :facepalm: